### DISCUSSION TITLE
Unable to Install 0.103

### DISCUSSION THREAD

**Discussion by Darren18L** [seerge/g-helper#993] [General]
I've downloaded and installed the newest update but it is still prompting mw to download 0.103. I've restated but that didn't solve the issue

**Comment by seerge** [comment]
@Darren18L then looks like you are downloading and running not 0.103 :) try to delete old one, and unzip new one over old spot + check "run on startup" after (to make sure you are scheduling this file and not something that you have unzipped into temp folder or so ...)
